php/libraries/LorisForm.class.inc

```diff
@@ -169,14 +169,19 @@ class LorisForm
     /**
      * Reimplementation of HTML_QuickForm's "addHidden" API.
      *
-     * @param string $name  The element name.
-     * @param string $value The element value.
+     * @param string $name       The element name.
+     * @param string $value      The element value.
+     * @param array  $attributes Additional html attributes for element
      *
      * @return &array
      */
-    public function addHidden($name, $value)
+    public function addHidden($name, $value, $attributes=array())
```

Comment:
@jstirling91 the calling method addElement() passes this parameter and sets it to default array() if nothing is passed. so the default is set higher up in the hierarchy but i added it for consistency